### DISCUSSION TITLE
Final Changes discussions

### DISCUSSION THREAD

**Discussion by Undaunted01** [Undaunted01/Esprela-Funa-Baconua#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JonKaicerBaconua** [comment]
CHANGE TA COLORS ASAP

**Reply by Undaunted01** [reply]
ok naman, clean naman interface ta

**Reply by JonKaicerBaconua** [reply]
ok naman to aton, dow nami lang mag improve sa mga colors kung makaya

**Reply by JohnLloydFuna** [reply]
may  code sina busy paman abi ko diri sa lola ko

**Reply by Undaunted01** [reply]
karon nalang?, kay gabi pa man deadline
free man ko karon

**Reply by JonKaicerBaconua** [reply]
Karon nalang gali, pwede man

**Reply by JohnLloydFuna** [reply]
san o ang deadline?

**Reply by JonKaicerBaconua** [reply]
Subong, ifree sched niyo, last clutch

**Reply by JonKaicerBaconua** [reply]
Karon 11:59 ang deadline

**Reply by Undaunted01** [reply]
payt karon kay free ko
gab.i pa man

**Reply by JonKaicerBaconua** [reply]
Payts karon ah

**Reply by Undaunted01** [reply]
ipang finalize nalang daad ang mga video

**Reply by Undaunted01** [reply]
oks man lang na iban ang interface na design, di man matandog ang major nga parts sa code ta

**Comment by Undaunted01** [comment]
MEETING @GMEET 4:00 PM

To join the video meeting, click this link: https://meet.google.com/oyd-wreb-fmx
Otherwise, to join by phone, dial +1 657-233-7746 and enter this PIN: 737 837 700#

**Reply by JonKaicerBaconua** [reply]
Yes boss

**Reply by JohnLloydFuna** [reply]
noted
